lib/kernel-manager.js

```diff
@@ -101,11 +100,7 @@ class KernelManager {
         break;
     }
 
-    if (projectPath != null) {
-      kernelStartDir = projectPath;
-    } else {
-      kernelStartDir = currentPath;
-    }
+    const kernelStartDir = projectPath != null ? projectPath : currentPath;
```

Comment:
👀  crazy javascript null handling.

Anyway, I think we should be fine since `projectPath` is either `null`, `undefined` or a proper path.